airflow/providers/ssh/hooks/ssh.py

```diff
@@ -305,7 +305,7 @@ def get_conn(self) -> paramiko.SSHClient:
         client.connect(**connect_kwargs)
 
         if self.keepalive_interval:
-            client.get_transport().set_keepalive(self.keepalive_interval)
+            client.get_transport().set_keepalive(self.keepalive_interval)  # type: ignore[union-attr]
```

Comment:
I have to use "ignore" here because of "paramiko" lib isn't well typed.
Maybe someone has any suggestions?

Reply:
I think just add the comment above it why the ignore. This is only a local problem that will nt bubble to any users of the provider.

Reply:
Added